python/lsst/ts/standardscripts/maintel/laser_tracker/shut_down.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ShutDown"]
+
+from lsst.ts import salobj
+from lsst.ts.idl.enums.LaserTracker import LaserStatus
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+class ShutDown(salobj.BaseScript):
+    """Shut down Laser Tracker.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+    - "Shutting down Laser Tracker.": Turning laser power off.
+    - "Waiting for laser Tracker to warm down.": Waiting Laser Tracker to shut down.
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(index, descr="Shut down laser tracker.")
+
+        self.config = None
+
+        self.laser_tracker = RemoteGroup(
+            domain=self.domain,
+            components=["LaserTracker:1"],
+            intended_usage=None if add_remotes else Usages.DryTest,
+            log=self.log,
+        )
+
+        self.timeout_laser_warmdown = 60
+        self.timeout_laser_power = 30
+        self.timeout_std = 60
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        self.config = config
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = self.timeout_laser_power + self.timeout_laser_warmdow
+
+    async def wait_laser_status_ok(self):
```

Comment:
should i be calling this method from the set_up and passing OFF as attribute?

Reply:
You should probably rename the method `wait_laser_status_off`... 

I was thinking about this. We may want to actually have a `LaserTracker` class in ts_observatory_control and put all this functionality there. 

But I don't know.. For now I think we can keep these.

Reply:
Done